client/components/jar/NFTGatedWithdrawalSection.tsx

```
@@ -312,55 +341,88 @@ export const NFTGatedWithdrawalSection: React.FC<
-							/>
+					<div className="space-y-3 rounded-lg border border-dashed border-[#f0e6d8] bg-white/60 p-4">
+						<p className="text-sm text-[#8b7355]">
+							Use manual entry if your NFT does not appear in the visual picker.
+							Provide the collection address, the exact token ID, and its token
+							standard so we can verify ownership before withdrawing.
+						</p>
+						<div className="grid gap-4 md:grid-cols-2">
+							<div className="md:col-span-2">
+								<label className="text-sm font-medium text-[#3c2a14]">
+									NFT Contract Address
+								</label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)